.concourse/tasks/upgrade.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e
+
+echo "Running the upgrade test"
+
+KUBECF_LATEST_RELEASE=$(cat kubecf-github-release/version)
+export KUBECF_LATEST_RELEASE
+export SCF_CHART="https://github.com/cloudfoundry-incubator/kubecf/releases/download/v${KUBECF_LATEST_RELEASE}/kubecf-bundle-v${KUBECF_LATEST_RELEASE}.tgz"
+
+export ENABLE_EIRINI=false
+export SCF_OPERATOR=true
+
+export FORCE_DELETE=true
+export HELM_VERSION="v3.1.1"
+export BACKEND=imported
```

Comment:
Instead of `BACKEND=imported make kubeconfig` it should be `BACKEND=ekcp make kubeconfig` everywhere, but that's outside of the scope of this PR.